include/picongpu/plugins/radiation/param.hpp

```diff
@@ -0,0 +1,28 @@
+/* Copyright 2024 Rene Widera
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "picongpu/defines.hpp"
+
+// clang-format off
+#    include "picongpu/param/radiation.param"
+#    include "picongpu/param/radiationObserver.param"
+#    include "picongpu/unitless/radiation.unitless"
+// clang-format on
```

Comment:
```suggestion
#include "picongpu/param/radiation.param"
#include "picongpu/param/radiationObserver.param"
#include "picongpu/unitless/radiation.unitless"
```
I'm aware that this a a copy-paste thing from another file but in doing so, this tiny formatting fix should be allowed.

Reply:
fixed